openslides/core/config.py

```diff
@@ -37,21 +49,42 @@ def __init__(self) -> None:
         self.config_variables = {}  # type: Dict[str, ConfigVariable]
 
         # Index to get the database id from a given config key
-        self.key_to_id = {}  # type: Dict[str, int]
+        self.key_to_id = None  # type: Optional[Dict[str, int]]
 
     def __getitem__(self, key: str) -> Any:
         """
         Returns the value of the config variable.
         """
-        # Build the key_to_id dict
-        self.save_default_values()
-
         if not self.exists(key):
             raise ConfigNotFound(_('The config variable {} was not found.').format(key))
 
         return CollectionElement.from_values(
             self.get_collection_string(),
-            self.key_to_id[key]).get_full_data()['value']
+            self.get_key_to_id()[key]).get_full_data()['value']
+
+    def get_key_to_id(self) -> Dict[str, int]:
+        """
+        Returns the key_to_id dict. Builds it, if it does not exist.
+        """
+        if self.key_to_id is None:
+            async_to_sync(self.build_key_to_id)()
```

Comment:
Maybe you have to ensure that all default values are in the database or in the cache before you can run this line.

Reply:
They are inserted in the database after each migration. If the elements are not in the cache, the cache is build with element_cache.get_all_full_data()